Tensile/KernelWriterAssembly.py

```diff
@@ -7311,7 +7313,7 @@ def globalReadGuardK(self, kernel, tP, vregSetIdx):
       tmpSgpr = self.getTmpSgpr(2).idx()
       maxAddrSgpr = tmpSgpr
 
-      kStr += self.comment1("flat addressing - max read address = Tensor2dSize%s"%tc)
+      kStr += self.comment1("global addressing - max read address = Tensor2dSize%s"%tc)
```

Comment:
Is global addressing identical to flat addressing? There are a few other spots in the code/comments that reference modifications for flat addresses that may at least need a comment update (if no code changes required):
- Line 1416
- Line 2576
- Line 4317

Reply:
global and flat are mostly identical.
global has one more operand, but we can disable it by putting "off".

Reply:
I updated above lines.